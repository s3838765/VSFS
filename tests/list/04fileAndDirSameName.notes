NOTES V1.0
@file1
=file1/
