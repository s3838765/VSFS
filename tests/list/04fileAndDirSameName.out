drwxrwxrwx 1 dib dib 512 Oct 21 23:30 file1/
-rwxrwxrwx 1 dib dib   0 Oct 21 23:30 file1
